src/node/build/buildPluginHtml.ts

```diff
@@ -67,11 +69,33 @@ export const createBuildHtmlPlugin = async (
     }
   }
 
+  const injectSystemJs = (html: string) => {
+    let tag: string
+    if (useSystemJs === false) {
+      return html
+    } else if (useSystemJs === true || useSystemJs === 'inline') {
+      const systemJsRuntime = fs.readFileSync(
+        path.resolve(__dirname, '../../../node_modules/systemjs/dist/s.min.js'),
```

Comment:
should use `require.resolve` here since the actual installed location varies by package manager